backend/app/endpoints/auth.py

```diff
@@ -44,7 +44,7 @@ def login(
         raise HTTPException(status_code=404, detail="User not found")
```

Comment:
It's better to have same error for checking email and pasword on login:
```suggestion
        raise HTTPException(status_code=401, detail="Invalid credentials")
```